tasks/validate.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # module errors validating
-module Validate
+module Error
```

Comment:
what about file name?
`module` creates a namespace and all error classes should be located in `errors folder`:
`/errors/field_error.rb`
`/errors/wrong_number_error.rb`
you can check zeitwerk gem documentation for more examples

Reply:
done. Have created Error folder and 2 files. I looked zeitwerk gem. i need some more explanation how to use it.